packages/sitecore-jss-rendering-host/package.json

```diff
@@ -42,15 +41,14 @@
     "@types/mocha": "^10.0.1",
     "@types/node": "^18.11.18",
     "@types/webpack": "5.28.0",
-    "@types/webpack-dev-middleware": "2.0.5",
-    "@types/webpack-dev-server": "3.1.7",
     "chai": "^4.3.0",
     "del-cli": "^5.0.0",
     "eslint": "^8.33.0",
     "mocha": "^10.2.0",
     "nyc": "^15.1.0",
     "ts-node": "^10.9.1",
-    "typescript": "~4.9.5"
+    "typescript": "~4.9.5",
+    "webpack-dev-server": "4.13.2"
```

Comment:
I assume that _webpack-dev-server_ should be removed, right?

Reply:
> I assume that webpack-dev-server should be removed, right?

It's actually needed, sample would fail at runtime without it, attempting to import 'compression' and 'express' without it.
The dependencies are kind of weird like that in the package. We import some things in code that are not directly referenced as dependencies.

Reply:
@art-alexeyenko But I see that you upgraded webpack-dev-server to the latest server, so it works fine anyway?